examples/05_browser_text_uptime_agent.py

```diff
@@ -0,0 +1,78 @@
+"""
+This example demonstrates how to use the @browser-text tool to fetch and analyze
+uptime data from API status pages. It shows how to:
+1. Use @browser-text to fetch status page content
+2. Extract uptime percentage for specific services
+3. Handle different status page formats
+"""
+
+import asyncio
+
+from pydantic import BaseModel, Field  # pyright: ignore [reportUnknownVariableType]
+
+import workflowai
+from workflowai import Model, Run
+
+
+class UptimeInput(BaseModel):
+    """Input for checking API uptime."""
+    status_page_url: str = Field(
+        description="URL of the status page to check",
+        examples=["https://status.openai.com", "https://status.anthropic.com"],
+    )
+    service_name: str = Field(
+        description="Name of the specific API service to check",
+        examples=["API", "Chat Completions", "Embeddings"],
+    )
+
+
+class UptimeOutput(BaseModel):
+    """Output containing uptime percentage."""
+    uptime_percentage: float = Field(
+        description="The uptime percentage for the specified service over the last 30 days",
+        examples=[99.99, 98.5],
+        ge=0.0,
+        le=100.0,
+    )
+
+
+@workflowai.agent(
+    id="uptime-checker",
+    model=Model.GPT_4O_MINI_LATEST,
+)
+async def check_uptime(input: UptimeInput, use_cache: str = "never") -> Run[UptimeOutput]:
```

Comment:
to answer you question, the name of the `input` parameter does not matter, it is only a positional argument.
So you can name it anything else. In python `_` seems to be the better option to express that it has no name, but `uptime_input` would work as well